pkg/networkmanager/interfaces.js

```diff
@@ -2596,6 +2612,19 @@ PageNetworkInterface.prototype = {
                                                        dev.DeviceType == 'bridge'));
         $('#network-interface-delete').toggle(is_deletable && managed);
 
+        function render_interface_section_separator(title) {
+            return $('<td class="network-interface-separator" colspan="100%">').text(title);
```

Comment:
This would be a `<td>` without a `<tr>` no?  I think that's why the "General Information" is 14px, while "Settings" is 16px.